### PR TITLE
URL management topic update (wildcards)

### PR DESCRIPTION
**Pull request by dabrt**
| Question      | Answer |
| ------------- | --- |
| Versions      | master |

Updated URL management topic to account for changes in URL wildcard workflow and changes in the UI.

### CODE REVIEW

File: docs/guide/url_management.md [hunk 1]
```diff
@@ -1,36 +1,39 @@
...
-The Link manager contains all the necessary information about each address including its status (valid or invalid)
-and the time it was last checked (when the system attempted to validate the URL).
-By default, all URLs are valid.
 
-You can use the `ezplatform:check-urls` command to check all the addresses stored in the URL table.
-It validates the links by accessing them one by one.
-If a broken link is found, its status is set to "invalid". The last checked field is always updated.
+## External URL validation
+
+You can manually validate all the addresses from the URL table, by executing the `ezplatform:check-urls` command. It validates the links by accessing them one by one and updates the value in the Last checked field. If a broken link is found, its status is set to "invalid".
```
Comment (DominikaK) [inline docs/guide/url_management.md:18]:
```suggestion
You can validate all the addresses from the URL table by executing the `ezplatform:check-urls` command. It validates the links by accessing them one by one and updates the value in the Last checked field. If a broken link is found, its status is set to "invalid".
```
The same command can be run automatically, so I'd get rid ot the "manually"

File: docs/guide/url_management.md [hunk 2]
```diff
@@ -1,36 +1,39 @@
...
 
 - `http`
 - `https`
 - `mailto`
 
-### Enable automatic URL validation
+### Enabling automatic URL validation
 
-To enable automatic URL validation, you need to set up cron to run the `ezplatform:check-urls` command periodically.
+To enable automatic URL validation, you set up cron to run the `ezplatform:check-urls` command periodically.
```
Comment (DominikaK) [inline docs/guide/url_management.md:28]:
```suggestion
To enable automatic URL validation, set up cron to run the `ezplatform:check-urls` command periodically.
```

File: docs/guide/url_management.md [hunk 3]
```diff
@@ -63,28 +65,29 @@ ezplatform:
                     	enabled: false
 ```
 
-Available options are protocol-specific.
+Available options are protocol-specific. For details, see the tables below.
```
Comment (DominikaK) [inline docs/guide/url_management.md:68]: Not really necessary, as the tables are immediately below this.

File: docs/guide/url_management.md [hunk 4]
```diff
@@ -63,28 +65,29 @@ ezplatform:
...
-| batch_size         | Maximum number of asynchronous requests                             | 10            |
-| ignore_certificate | Verify the peer's SSL certificate / certificate's name against host | false         |
+| Option             | Description                                                                              |  Default value |
+|----------------------|----------------------------------------------------------------------------------------|-----------------------|
+| enabled            | Enables   link validation.                                                      | true          |
+| timeout            | Defines the time that the request is allowed to take (in seconds).                       | 10            |
+| connection_timeout | Defines the time that the connect phase is allowed to take (in seconds).                 | 5             |
+| batch_size         | Defines a maximum   number of asynchronous requests.                                     | 10            |
+| ignore_certificate | Decides if the peer's SSL certificate or the certificate's is verified against the host. | false         |
```
Comment (DominikaK) [inline docs/guide/url_management.md:78]:
```suggestion
| ignore_certificate | Decides if the peer's SSL certificate or the certificate name are verified against the host. | false         |
```

File: docs/guide/url_management.md [hunk 5]
```diff
@@ -94,7 +97,7 @@ namespace eZ\Bundle\EzPublishCoreBundle\URLChecker;
 interface URLHandlerInterface
 {
     /**
-     * Validates given list of URLs.
+     * Validates a given list of URLs.
```
Comment (DominikaK) [inline docs/guide/url_management.md:100]:
```suggestion
     * Validates given list of URLs.
```
This is part of product code

File: docs/guide/url_management.md [hunk 6]
```diff
@@ -119,11 +122,64 @@ app.url_checker.handler.custom:
...
+The `scheme` attribute is mandatory and has to correspond to the name of the protocol, for instance, `ftp`.
 
-## URL alias patterns
 
-You can configure how eZ Platform generates URL aliases. The configuration is available under `ezplatform.url_alias.slug_converter`, for example:
+## URL wildcards
+
+With wildcards you have a simple way to display content under a specific URL address, for example, when you decide to reorganize the architecture and want to protect users from losing easy access to the content that they have bookmarked.
+
+For each wildcard you can decide whether the user will see the updated content at the original address (direct type), or will be redirected to the new address (forward type).
```
Comment (DominikaK) [inline docs/guide/url_management.md:132]:
```suggestion
For each wildcard you can decide whether the user sees the updated content at the original address (Direct type), or is redirected to the new address (Forward type).
```

File: docs/guide/url_management.md (hunk 1 above)
Comment (dabrt) [reply docs/guide/url_management.md:18]: Automatic validation is mentioned below, hence the word "manually" to underline the difference between the two processes.

File: docs/guide/url_management.md (hunk 5 above)
Comment (dabrt) [reply docs/guide/url_management.md:100]: Is it how the inline comments look in the product or is it an example with comments added for clarity?

File: docs/guide/url_management.md [hunk 7]
```diff
@@ -1,36 +1,39 @@
 # URL management
 
-## External URL validation
+Manage external URL addresses and URL wildcards with a comfortable user interface that is available in the Back Office, Admin tab, the URL Manager node. Configure URL aliases to have human-readable URL addresses throughout your system.
```
[the remark this answers is not in the document]
Comment (dabrt) [reply docs/guide/url_management.md:3]: I thought the features called for a bit of introduction. The only explanation on what URL management is can be found in the release notes, which are not permanent.
That said, I would rather leave the intro here.

File: docs/guide/url_management.md (hunk 5 above)
Comment (DominikaK) [reply docs/guide/url_management.md:100]: This is an exact quote from code, see https://github.com/ezsystems/ezplatform-kernel/blob/master/eZ/Bundle/EzPublishCoreBundle/URLChecker/URLHandlerInterface.php#L12

File: docs/guide/url_management.md (hunk 7 above)
Comment (DominikaK) [inline docs/guide/url_management.md:3]: Fair enough, but you will need to duplicate/repeat the same information in the user guide as well. I'd suggest a bit more adaptation:
```suggestion
You can manage external URL addresses and URL wildcards in the Back Office, **Admin** tab, the **URL Manager** node. Configure URL aliases to have human-readable URL addresses throughout your system.
```

File: docs/guide/url_management.md [hunk 8]
```diff
@@ -142,46 +160,80 @@ ezplatform:
...
 
-- `--iteration-count` — to define how many Locations should be processed at once to reduce memory usage
-- `--location-id` — to regenerate URLs for specific Locations only, e.g. `ezplatform:urls:regenerate-aliases --location-id=1 --location-id=2`
+- `--iteration-count` — Defines how many Locations are processed at once to reduce memory usage
+- `--location-id` — Regenerates URL addresses for specific Locations only, e.g. `ezplatform:urls:regenerate-aliases --location-id=1 --location-id=2`
 
 ## URL wildcards
 
-Using the Public API you can set up global URL wildcards for redirections.
+With wildcards, you have a simple way to change the URL address for many content items at the same time, by replacing a portion of the destinaton's URL address.
```
Comment (DominikaK) [inline docs/guide/url_management.md:204]:
```suggestion
With wildcards, you can change the URL address for many Content items at the same time, by replacing a portion of the destination's URL address.
```

File: docs/guide/url_management.md [hunk 9]
```diff
@@ -142,46 +160,80 @@ ezplatform:
...
+    url_wildcards:
+        enabled: true
+```
+
+### Configuring URL wildcards in the Back Office
+
+The URL wildcards tab contains all the information about each URL wildcard. You can delete or modify existing entries, or create new ones.
+
+When you edit an existing wildcard URL or add a new one, you define the wildcard pattern and its destination.
+Also, you decide whether the user should see the updated content at the existing address (direct type), or should be redirected to the new address (forward type).
```
Comment (DominikaK) [inline docs/guide/url_management.md:227]: This information is repeated

File: docs/guide/url_management.md [hunk 10]
```diff
@@ -1,36 +1,46 @@
 # URL management
 
-## External URL validation
+You can manage external URL addresses and URL wildcards in the Back Office, **Admin** tab, the **URL Manager** node.
```
Comment (DominikaK) [inline docs/guide/url_management.md:3]:
```suggestion
You can manage external URL addresses and URL wildcards in the Back Office, **Admin** tab, the **URL Management** node.
```

File: docs/guide/url_management.md [hunk 11]
```diff
@@ -64,27 +73,30 @@ ezplatform:
...
+| ignore_certificate | Decides if the peer's SSL certificate or the certificate name are verified against the host. | false         |
 
 #### mailto protocol
 
 | Option             | Description                                                         | Default value |
 |--------------------|---------------------------------------------------------------------|---------------|
-| enabled            | Enables the validation                                              | true          |
+| enabled            | Enables link validation.                                            | true          |
+
+For more information about configuration, see [Configuration](configuration).
```
Comment (michal-myszka) [inline docs/guide/url_management.md:94]: Note: This link is not valid now (404)

**Pull request merged**